README.md

```diff
@@ -6,24 +6,95 @@
 
 flannel is a virtual network that gives a subnet to each host for use with container runtimes.
 
-Platforms like Kubernetes assume that each container (pod) has a unique, routable IP inside the cluster.
-The advantage of this model is that it reduces the complexity of doing port mapping.
+Platforms like Kubernetes assume that each container (pod) has a unique, routable IP inside the cluster. The advantage of this model is that it reduces the complexity of doing port mapping.
 
 ## Theory of Operation
 
-flannel runs an agent, flanneld, on each host and is responsible for allocating a subnet lease out of a preconfigured address space.
-flannel uses [etcd](https://github.com/coreos/etcd) to store the network configuration, allocated subnets, and auxiliary data (such as host's IP).
-The forwarding of packets is achieved using one of several strategies that are known as backends.
-The simplest backend is `udp` and uses a TUN device to encapsulate every IP fragment in a UDP packet, forming an overlay network.
+Flannel runs an agent, `flanneld`, on each host and is responsible for allocating a subnet lease out of a preconfigured address space. Flannel uses [etcd][etcd] to store the network configuration, allocated subnets, and auxiliary data (such as host's IP). Packets are forwarded through one of several [backend mechanisms][backends].
+
 The following diagram demonstrates the path a packet takes as it traverses the overlay network:
 
 ![Life of a packet](./packet-01.png)
 
-## Documenation
-- [Running](Documentation/running.md)
+## Getting started
+
+It's easy to use flannel with Kubernetes. Flannel can use the Kubernetes API as its backing store, meaning there's no need to deploy a discrete `etcd` cluster for `flannel`. This `flannel` mode is known as the *kube subnet manager* and is demonstrated below.
+
+### Adding flannel
+
+Flannel can be added to any existing Kubernetres cluster. It's simplest to add `flannel` before any pods using the pod network have been started.
+
+A great way to create a new Kubernetes cluster is with [kubeadm][kubeadm] which helpfully points out the need to define *a pod network CIDR* for example by passing in `--pod-network-cidr 10.244.0.0/16` when running `kubeadm init`. This pod network CIDR is the key prerequisite for using flannel with the kube subnet manager. If you're running flannel through some other means, then the pod CIDR might be set automatically for you or you can set it manually by passing [--pod-cidr][pod-cidr] to each of your kubelets.
```

Comment:
I cannot find a good reference page for simply k8s + flannel, but we might call this one out somewhere: https://kubernetes.io/docs/setup/pick-right-solution/